### PR TITLE
feat: add OpenTracing support

### PR DESCRIPTION
**Pull request by jahtalab**
add a separate bundle for opentracing
minor fixes

Requires https://github.com/elastic/apm-agent-js-core/pull/23 to be released!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "run-script": "node ./scripts/run-script.js",
     "test": "npm run lint && npm run build && npm run buildE2eBundles && npm run runNodeTests && npm run runUnitTests && npm run run-e2e && npm run build-docs",
     "lint": "standard",
-    "format": "prettier-standard src/**/*.js test/**/*.js"
+    "format": "prettier-standard 'src/**/*.js' test/**/*.js"
```
Comment (vigneshshanmugam) [inline package.json]: is this required? It works without quotes for me.

File: test/e2e/standalone-html/index.html [hunk 2]
```diff
@@ -9,13 +9,16 @@
   <body>
     <div id="app"></div>
     <script src="/dist/bundles/elastic-apm-js-base.umd.min.js"></script>
+    <script src="/dist/bundles/elastic-apm-opentracing.umd.min.js"></script>
```
Comment (vigneshshanmugam) [inline test/e2e/standalone-html/index.html]: are we planning to release opentracing support as separate script instead?

File: src/opentracing/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+function createTracer (apmBase) {
+  return createElasticTracer(apmBase.serviceFactory)
+}
+
+if (window && window.elasticApm) {
+  window.elasticApm.createTracer = createTracer.bind(window.elasticApm, window.elasticApm)
+}
+
+module.exports = {
```
Comment (vigneshshanmugam) [inline src/opentracing/index.js]: we dont need to do this, build script takes care of it.

File: package.json (hunk 1 above)
Comment (jahtalab) [reply package.json]: without quotes it doesn't format the top level files `/src/*.js`

File: test/e2e/standalone-html/index.html (hunk 2 above)
Comment (jahtalab) [reply test/e2e/standalone-html/index.html]: I think that's a better approach, considering the feature might not be used in a lot of use-cases. I would prefer to keep our main bundle size small.

What do you think?

**Comment by jahtalab** [comment]
@vigneshshanmugam , Would you please have another look at this one?
The docs PR is: https://github.com/elastic/apm-agent-js-base/pull/140

**Pull request merged**